src/Evolution/Systems/ScalarWave/Equations.hpp

```diff
@@ -97,21 +98,123 @@ struct ComputeNormalDotFluxes {
                         interface_unit_normal) noexcept;
 };
 
+/*!
+ * \ingroup NumericalFluxesGroup
+ * \brief Computes the penalty flux
+ *
+ * The penalty flux is given by:
+ *
+ * \f{align}
+ * G(\Psi) &= F(\Psi) = 0 \\
+ * G(\Pi) &= (n_i F^i(\Pi))_{\mathrm{int}} +
+ *     \frac{1}{2} p \left(v^{-}_{\mathrm{int}} - v^{+}_{\mathrm{ext}}\right) \\
+ * G(\Phi_i) &= (n_i F(\Phi_i))_{\mathrm{int}} - \frac{1}{2} p \left(
+ *      (n_i v^{-})_{\mathrm{int}} + (n_i v^{+})_{\mathrm{ext}}\right)
+ * \f}
+ *
+ * where \f$G\f$ is the interface normal dotted with numerical flux, the
+ * first terms on the RHS for \f$G(\Pi)\f$ and \f$G(\Phi_i)\f$ are the fluxes
+ * dotted with the interface normal (computed in
+ * ScalarWave::ComputeNormalDotFluxes), and \f$v^{\pm}\f$ are outgoing and
+ * incoming characteristic fields of the system (see characteristic_fields() for
+ * their definition). The penalty factor is chosen to be \f$p=1\f$.
+ */
+template <size_t Dim>
+struct PenaltyFlux {
+ private:
+  struct NormalTimesVPlus {
+    using type = tnsr::i<DataVector, Dim, Frame::Inertial>;
+    static std::string name() noexcept { return "NormalTimesVPlus"; }
+  };
+  struct NormalTimesVMinus {
+    using type = tnsr::i<DataVector, Dim, Frame::Inertial>;
+    static std::string name() noexcept { return "NormalTimesVMinus"; }
+  };
+
+ public:
+  using options = tmpl::list<>;
+  static constexpr OptionString help = {
+      "Computes the penalty flux for a scalar wave system. It requires no "
+      "options."};
+  static std::string name() noexcept { return "Penalty"; }
```

Comment:
Is there a particular reason why this is a different name than the name of the struct? I'm not asking for a change, just trying to understand...

Reply:
It seemed cleaner to me to remove the `Flux` postfix from `name()` so that input files read:

```
NumericalFlux:
  Penalty:  
```

instead of 

```
NumericalFlux:
  PenaltyFlux: 
```

where in the latter the postfix clearly seems redundant.